sdk/aqueduct/integrations/connect_config.py

```diff
@@ -147,18 +147,37 @@ class SlackConfig(BaseConnectionConfig):
     enabled: bool
 
 
-class AWSConfig(BaseConnectionConfig):
-    access_key_id: str
-    secret_access_key: str
-
-
 class _SlackConfigWithStringField(BaseConnectionConfig):
     token: str
     channels_serialized: str
     level: str
     enabled: str
 
 
+class DynamicK8sConfig(BaseConnectionConfig):
+    keepalive: Optional[str]
+    cpu_node_type: Optional[str]
+    gpu_node_type: Optional[str]
+    min_cpu_node: Optional[str]
+    max_cpu_node: Optional[str]
+    min_gpu_node: Optional[str]
+    max_gpu_node: Optional[str]
+
+
+class AWSConfig(BaseConnectionConfig):
+    access_key_id: str
+    secret_access_key: str
+    region: str
+    k8s: Optional[DynamicK8sConfig]
+
+
+class _AWSConfigWithStringField(BaseConnectionConfig):
+    access_key_id: str
+    secret_access_key: str
+    region: str
+    k8s_serialized: Optional[str]
```

Comment:
Can we document what type of serialization we are using? And what this is serializing.